MyHeartCounts/Onboarding/ComprehensionScreening.swift

```diff
@@ -71,3 +71,11 @@ private struct Question: ScreeningComponent {
         data.comprehension[keyPath: storage] == true
     }
 }
+
+
+#Preview {
```

Comment:
@lukaskollmer Would be great if we can add Previews to all major views to make some of the UI refactoring easier to work on.

Reply:
yeah i have to admit i never got into using previews (mainly bc they're just super slow and annoying to work with), that's why we don't really have any.

will try to be better about this going forward